src/devices/Bmi088/README.md

```diff
@@ -0,0 +1,22 @@
+# Bosch Bmi088 6-Axis Accelerometer & Gyroscope
+The Bosch Bmi088 6-Axis Accelerometer & Gyroscope is a 6 DoF (degrees of freedom) High-performance Inertial Measurement Unit (IMU).
+This sensor is a high-performance IMU with high vibration suppression. The 6-axis sensor combines a 16 bit triaxial gyroscope and a 16 bit triaxial accelerometer.
+
```

Comment:
You can mention in the description you've been using the Grove version if you want.
Please mention that the implementation is I2C only. The sensor supports SPI as well but it's not implemented